main.py

```diff
@@ -0,0 +1,161 @@
+import json
+import os
+from datetime import datetime, timezone
+from typing import Mapping, Tuple
+from uuid import UUID
+
+import jwt
+from flask import Request
+from requests import Session
+from requests.exceptions import RequestException
+
+from exceptions import InvalidNotifyKeyError, InvalidRequestError
+
+
+def log_info(**kwargs):
+    print(json.dumps({**kwargs, "severity": "INFO"}))
+
+
+def log_error(**kwargs):
+    print(json.dumps({**kwargs, "severity": "ERROR"}))
+
+
+def _create_jwt_token(secret: str, client_id: str) -> str:
+    headers = {"typ": "JWT", "alg": "HS256"}
+    claims = {"iss": client_id, "iat": int(datetime.now(timezone.utc).timestamp())}
+    return jwt.encode(payload=claims, key=secret, headers=headers)
+
+
+def create_notify_token(key: str) -> str:
+    service_id = key[-73:-37]
+    secret_key = key[-36:]
+
+    if not _is_valid_uuid(service_id):
+        raise InvalidNotifyKeyError("Service ID is not a valid uuid")
+
+    if not _is_valid_uuid(secret_key):
+        raise InvalidNotifyKeyError("API key is not a valid uuid")
+
+    return _create_jwt_token(secret_key, service_id)
+
+
+def _is_valid_uuid(identifier: str) -> bool:
+    try:
+        UUID(identifier, version=4)
+    except ValueError:
+        return False
+
+    return True
+
+
+NOTIFY_API_KEY = os.environ["NOTIFY_API_KEY"]
+
+NOTIFY_BASE_URL = "https://api.notifications.service.gov.uk/v2"
+
+template_id_mapping = {
+    ("HH", "GB-ENG", "en"): "0c5a4f95-bfa4-4364-9394-8499b4d777d5",
+    ("HH", "GB-WLS", "en"): "0c5a4f95-bfa4-4364-9394-8499b4d777d5",
+    ("HH", "GB-WLS", "cy"): "755d73d1-0cb6-4f2f-95e9-857a2ad071bb",
+    ("HH", "GB-NIR", "en"): "0889cfa1-c0eb-4ba6-93d9-acc41b060152",
+    ("HH", "GB-NIR", "ga"): "0889cfa1-c0eb-4ba6-93d9-acc41b060152",
+    ("HH", "GB-NIR", "eo"): "0889cfa1-c0eb-4ba6-93d9-acc41b060152",
+    ("HI", "GB-ENG", "en"): "71de56dc-f83b-4899-93ab-7fe61e417c2e",
+    ("HI", "GB-WLS", "en"): "71de56dc-f83b-4899-93ab-7fe61e417c2e",
+    ("HI", "GB-WLS", "cy"): "1001ac43-093d-425c-ac7d-68df5147c603",
+    ("HI", "GB-NIR", "en"): "ed1c2e9f-c81e-4cc2-889c-8e0fa1d2ce1b",
+    ("HI", "GB-NIR", "ga"): "ed1c2e9f-c81e-4cc2-889c-8e0fa1d2ce1b",
+    ("HI", "GB-NIR", "eo"): "ed1c2e9f-c81e-4cc2-889c-8e0fa1d2ce1b",
+    ("CE", "GB-ENG", "en"): "4077d2cf-81cd-462d-9065-f227a7c39a8d",
+    ("CE", "GB-WLS", "en"): "4077d2cf-81cd-462d-9065-f227a7c39a8d",
+    ("CE", "GB-WLS", "cy"): "e4a4ebea-fcc8-463b-8686-5b8a7320f089",
```

Comment:
The `form_type` values in the schemas (and so passed from runner) are `H`, `I` and `C` e.g. https://github.com/ONSdigital/eq-questionnaire-schemas/blob/master/source/jsonnet/england-wales/census_household.jsonnet#L141

Reply:
Done